oak-core/src/main/java/org/apache/jackrabbit/oak/query/SQL2Parser.java

```diff
@@ -636,6 +636,7 @@ private ConstraintImpl parseConditionFunctionIf(String functionName) throws Pars
                 }
             }
         } else if ("NATIVE".equalsIgnoreCase(functionName)) {
+            LOG.warn("native queries are deprecated. query:{}", statement);
```

Comment:
nitpick: 
```suggestion
            LOG.warn("native queries are deprecated. query: {}", statement);
```
Is it a convention that the log messages start with lower case? And even if that is the case, maybe the second sentence part should start with upper case.

Reply:
Done